src/main/java/Incy.java

```diff
@@ -0,0 +1,77 @@
+import java.util.Scanner;
+
+public class Incy {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        String input;
+        Task[] tasks = new Task[100];
```

Comment:
Please avoid the use of Magic Numbers. Instead declare a constant with the value you deem is right, and use that constant.

Reference: https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers